### DISCUSSION TITLE
Getting `name 'httpd' is not defined` with `ipv6.disable=1`

### DISCUSSION THREAD

**Discussion by SinTan1729** [fish-shell/fish-shell#9126] [Q&A] [answered]
It was discussed in the issue #3857 but the solution given there is to change `ipv6_disable=1` to `ipv6.disable_ipv6=1` in `/etc/default/grub`. It does work.

The issue is that it doesn't prevent `ipv6` leaks, as tested in [ipleak.net](https://ipleak.net).

**Comment by faho** [comment, marked as answer]
Turns out we can just check errno.

Note: I do not recommend turning off ipv6.
